### DISCUSSION TITLE
Okey, I reverted my different IO, so you can fix compy paste and import chips

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#62] [General]
I said all in title...

**Comment by firecerne** [comment]
As you did not completly revert it, I am trying to figure out how the signal type changes when you select a different one in the context menu so I can fix it. Can you help me ?

**Reply by UkrainianBanderasCat** [reply]
I think I forget to remove the menu to select Output and Input. I will do different IO Sometime in the future

**Reply by firecerne** [reply]
No, it's fine. I just need to know what made the menu change the pin type to be able to finish fixing it

**Reply by UkrainianBanderasCat** [reply]
It do nothing, It is just UI

**Reply by firecerne** [reply]
yes but , if I remember correctly, it did something in the past ?

**Reply by UkrainianBanderasCat** [reply]
Yes, but I remove that script, so now it do nothing

**Reply by firecerne** [reply]
Yes , but what was that script before ?

**Reply by UkrainianBanderasCat** [reply]
The script was Signal, but to implement it, you need to rewrite Save System
